content/en/docs/contribute/start.md

```diff
@@ -210,15 +210,11 @@ to base your work on. Use these guidelines to make the decision:
     periodically merge these to `master`. This kind of branch has a name like
     dev-\<version>-\<language code>.\<team milestone>; for example:
     `dev-{{< release-branch >}}-ja.1`.
-- If you're writing or updating documentation for a feature change release,
-  then you need to know the major and minor version of Kubernetes that
-  the change will first appear in.
-  - For example, if the feature gate JustAnExample is going to move from alpha
```

Comment:
I trimmed some of the content here because I felt like it was additional complexity with little gain for the reader.

I'm open to alternative phrasing however.

Reply:
The verbosity here is to help people coming from a [k/k](https://github.com/kubernetes/kubernetes) workflow who target _master_ when documenting chnages that go live in the next minor release.